homeassistant/components/ecobee/__init__.py

```diff
@@ -63,6 +72,16 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
+    hass.async_create_task(
+        discovery.async_load_platform(
+            hass,
+            Platform.NOTIFY,
+            DOMAIN,
+            {CONF_NAME: entry.title, ATTR_CONFIG_ENTRY_ID: entry.entry_id},
+            hass.data[DATA_ECOBEE_CONFIG],
```

Comment:
The whole original config needs to be passed, not only the ecobee config.

Reply:
Fixed in new commit